uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
```

Comment:
Minor note: there is a subtle difference between the following two functions:

```python
def f(x=5):
    print(x)

def f(x=None):
    x = x or 5
    print(x)
```

The difference is that oftentimes we'll apply arguments like ``f(**d)`` and ``d`` will be a dictionary like ``{'x': None}`` and in that case the two functions will print differently.

Can you do a quick ``grep`` of the codebase for all of the places where ``session.assign_badges`` is called to make sure this isn't the case anywhere?  Assuming it's not, then this should be good to merge.  I just want to make sure that this cleanup doesn't subtly break something.

(Relatedly: in a case where we're relying on a behavior this subtle, we should ALWAYS leave a code comment explaining why.  Since I'm the one who write this function originally, I'm definitely the one to blame for not doing that, assuming anything like this is the case.)